### DISCUSSION TITLE
Is it possible to access the vertices of a mesh.

### DISCUSSION THREAD

**Discussion by holycowbatmann** [gen2brain/raylib-go#357] [General]
Is it possible to access the vertices of a mesh? Mesh.Vertices is a float...

https://pkg.go.dev/github.com/gen2brain/raylib-go/raylib#Mesh

**Comment by gen2brain** [comment]
`unsafe.Slice(somemesh.Vertices, 3)` will return []float32.

**Reply by holycowbatmann** [reply]
unsafe.Slice(somemesh.Vertices, somemesh.VertexCount *3) would give me the full slice right?

**Reply by holycowbatmann** [reply]
Its working perfectly. thank you
